smart-contracts/scripts/deployments/governor.js

```diff
@@ -30,8 +30,9 @@ async function main() {
     'UnlockProtocolTimelock'
   )
 
-  // one week in seconds
-  const MINDELAY = 60 * 24 * 7
```

Comment:
poups! I forgot about that one/ We'll have to think about ways to tell everyone we fudged it.